cmd/commands/cli/command.go

```diff
@@ -702,3 +709,54 @@ func parseStartFlags(serviceType string, args ...string) (service.Options, confi
 
 	return nil, config.ServicesOptions{}, errors.New("service type not found")
 }
+
+func (c *cliApp) servicePrice(argsString string) {
+	const paymentsHelp = "Usage: service price <openvpn|wireguard> [gb|minute] [value]"
```

Comment:
Quite a strange flow to set a price and then to call another command `service start`. Lets be consistent and pass service create variables while starting it. Same as we do all other service variables `service start openvpn --access-policy.list=mysterium --shaper.enabled --price-minute=3000`

do same as shared configs `access-policy.list`

Reply:
Removed it, now it goes through the arguments for service start.